indico/modules/events/persons/client/js/EmailDialog.jsx

```diff
@@ -206,7 +231,11 @@ export function EmailDialog({
       validate={validate}
     >
       {({submitSucceeded}) =>
-        submitSucceeded ? <EmailSentMessage count={sentEmailsCount} /> : form
+        submitSucceeded ? (
+          <EmailSentMessage count={sentEmailsCount} warningMessage={sentEmailsWarning} />
```

Comment:
Maybe we could just do this and keep `EmailSentMessage` as is to keep the diff smaller?
```suggestion
          <>
            <EmailSentMessage count={sentEmailsCount} warningMessage={sentEmailsWarning} />
            <Message warning visible icon="warning sign" content={sentEmailsWarning} />
          </>
```